antsibull/data/docsite/role.rst.j2

```diff
@@ -0,0 +1,290 @@
+                    {% elif value['default'] is not none and value['default'] is sameas false %}
+                        {% set _x = value.update({'default': 'no'}) %}
+                    {% endif %}
+                    {% if value['type'] == 'bool' %}
+                        {% set _x = value.update({'choices': ['no', 'yes']}) %}
+                    {% endif %}
+                    {# Show possible choices and highlight details #}
+                    {% if value['choices'] %}
+                        <ul style="margin: 0; padding: 0"><b>Choices:</b>
+                            {% for choice in value['choices'] %}
+                                {# Turn boolean values in 'yes' and 'no' values #}
+                                {% if choice is sameas true %}
+                                    {% set choice = 'yes' %}
+                                {% elif choice is sameas false %}
+                                    {% set choice = 'no' %}
+                                {% endif %}
+                                {% if (value['default'] is not list and value['default'] == choice) or (value['default'] is list and choice in value['default']) %}
+                                    <li><div style="color: blue"><b>@{ choice | escape }@</b>&nbsp;&larr;</div></li>
+                                {% else %}
+                                    <li>@{ choice | escape }@</li>
+                                {% endif %}
+                            {% endfor %}
+                        </ul>
+                    {% endif %}
+                    {# Show default value, when multiple choice or no choices #}
+                    {% if value['default'] is not none and value['default'] not in value['choices'] %}
+                        <b>Default:</b><br/><div style="color: blue">@{ value['default'] | tojson | escape }@</div>
+                    {% endif %}
+                </td>
+                {# description #}
+                <td>
+                    {% for desc in value['description'] %}
+                        <div>@{ desc | replace('\n', '\n    ') | html_ify }@</div>
+                    {% endfor %}
+                    {% if value['aliases'] %}
+                        <div style="font-size: small; color: darkgreen"><br/>aliases: @{ value['aliases']|join(', ') }@</div>
+                    {% endif %}
+                </td>
+            </tr>
+            {% if value['options'] %}
+                @{ loop(value['options']|dictsort) }@
+            {% endif %}
+        {% endfor %}
+    </table>
+    <br/>
+{%   endif %}
+
+.. Notes
+
+{%   if ep_doc['notes'] -%}
+Notes
+^^^^^
+
+.. note::
+{%     for note in ep_doc['notes'] %}
+   - @{ note | rst_ify | indent(width=5) }@
+{%     endfor %}
+{%   endif %}
+
+.. Seealso
+
+{%   if ep_doc['seealso'] -%}
+See Also
+^^^^^^^^
+
+.. seealso::
+
+{%     for item in ep_doc['seealso'] %}
+{%       if item.module is defined and item.description %}
```

Comment:
I don't know that any of the example roles are exercising this bit of the code yet. Is there a way to 'dummy up' some See Also sections so we can be sure this part works?

Reply:
Here's now an example: https://ansible.fontein.de/collections/felixfontein/acme/acme_certificate_role.html#see-also